data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Stormhold Artifacts [1] Coded Message"
+	name "Go to <planet>"
+	description `Head to <planet> to deal with a "security issue."`
+	landing
+	to offer
+		has "event: stormhold korath issue"
+		not "Hai Reveal [B12] Devil-Hide Battle: offered"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai" "Hai (Unfettered)" "Pirate"
+		not planet Stormhold
+	destination Stormhold
+	on offer
+		conversation
+			`As soon as you land, your communications console beeps urgently. It's a coded message from <planet> that simply reads, "Security issue. Return to <planet> as soon as possible."`
+			`	The digital signature claims to be of diplomatic origin, and from the burning acorn insignia, it is most likely from the Unfettered.`
+				accept
+
+mission "Korath Stormhold Artifacts [2] Drones"
+	landing
+	name "Destroy Korath drones at <waypoints>."
+	description `Destroy Korath stealth drones at <waypoints>. This is an optional mission; an elite Navy group will do it if you don't.`
+	source Stormhold
+	destination Stormhold
+	waypoint Almach
+	to offer
+		has "Korath Stormhold Artifacts [1] Coded Message: done"
+		not "Hai Reveal [B12] Devil-Hide Battle: offered"
+	on offer
+		log "Factions" "Korath" `According to the Unfettered, the Korath were intelligent and well-cultured reptiles, until their civilization fell apart a few centuries ago. They peacefully coexisted with the Hai until they got desperate for resources and invaded. The Hai valiantly defended themselves against the Korath, but eventually the Korath nearly wiped themselves out through infighting. The few survivors live east of Syndicate space and frequently raid nearby human settlements.`
+		log `There was an abundance of Korath artifacts at Stormhold, but Batu fears Stormhold was merely a staging area and sales distribution site. He expects this is part of a larger Korath outfit sales operation that is based on a nearby planet.`
+		conversation
+			`Choot'k greets you, with a worried-looking Batu at his side. "Welcome back, Captain <last>! We found many spoils of war on this planet. Sadly, there weren't any jump drives," he says, winking. "The pirates had plenty of other gadgetry littering their bunkers, including limited-access human goods and Hai equipment. We've found a few Korath outfits, too, including some quite advanced ones. Navy inventory crewmen are gathering it to secure areas for inventory and analysis before divvying it up."`
```

Comment:
```suggestion
			`Choot'k greets you, with a worried-looking Batu at his side. "Welcome back, Captain <last>! We found many spoils of war on this planet. Sadly, there weren't any jump drives," he says, winking. "The pirates had plenty of other gadgetry littering their bunkers, including limited-access human goods and Hai equipment. We've found a few Korath outfits, too, including some quite advanced ones. It looks like these pirates must have gotten lucky and managed to actually loot one of their ships! Navy inventory crewmen are gathering it to secure areas for inventory and analysis before divvying it up."`
```

Just a small extra sentence to convey that pirates looting a Korath ship is a really impressive and rare thing; and to establish that this is all potentially just from a single ship. This is in part to counter the player's perception of things vs the storytelling detail. A Palavret carries up to 250 people, with 777 tons of outfits, and the ship itself is another 720. That's a huge amount of stuff. But a player can look at a Palavret and go "Eh, that's only 21 outfits (not counting rifles or ammunition). That's not much stuff. So if this base has *lots* of stuff, they must have captured lots of ships!" Not realizing that the story is looking at *all* the artifacts coming off a ship, not just the big individual outfits that the player cares about. Counting all the personal effects and things that are items but not outfits, a Palavret probably has tens of thousands of parts. 

(For comparison, the average car has about 30k parts; and if you count all the nuts and bolts and things like those as just being part of larger items, the average car still has about 1800 individual parts. So something like the Palavret... Yeah, probably actually in the millions of parts...